test/ddprof_pprof-ut.cc

```diff
@@ -45,7 +45,7 @@ void test_pprof(const DDProfPProf *pprofs) {
 
   // Check that encoded time is close to now
   time_t local_time = time(NULL);
-  EXPECT_TRUE(local_time - start.seconds < 1);
+  EXPECT_TRUE(local_time - start.seconds < 2);
```

Comment:
Ah, right.  Since `time()` only returns second-level granularity, if the true time is, say, `x.999` seconds (truncated to `x`), then the subsequent time may be, say `(x+1).001` so even though the absolute times are quite close, the second-by-second epochs will not pass (`1` is not greater than `(x+1) - x`).

Since we record time in the pprof at a greater granularity than seconds, it may also be possible to use a different time interface.  I have no idea what an appropriate tolerance is; for the time being, I feel that this check is OK.